src/main/java/uk/gov/hmcts/reform/pip/account/management/service/AuditService.java

```diff
@@ -33,8 +37,29 @@ public AuditService(AuditRepository auditRepository) {
      * @param pageable The pageable object to query by.
      * @return Returns the audit logs in a page.
      */
-    public Page<AuditLog> getAllAuditLogs(Pageable pageable) {
-        return auditRepository.findAllByOrderByTimestampDesc(pageable);
+    public Page<AuditLog> getAllAuditLogs(Pageable pageable, String email, String userId,
+        List<AuditAction> auditActions, LocalDateTime filterStartDate, LocalDateTime filterEndDate) {
+
+        // If user provenance id is supplied then find by an exact match
```

Comment:
We are not filtering by user provenance ID, just user ID

Reply:
fixed.